web-wallet/src/lib/components/Transactions/Transactions.svelte

```diff
@@ -36,53 +48,46 @@
 
 <article in:fade|global class="transactions">
 	<header class="transactions__header">
-		<slot name="heading"/>
+		<h3 class="h4">Transactions</h3>
 	</header>
 
-	<div class="transactions__lists">
-		{#if transactions.length}
-			{#each transactions as transaction (transaction.id)}
-				<dl class="transactions-list">
-					<dt class="transactions-list__term">Hash</dt>
-					<dd class="transactions-list__datum transactions-list__datum--hash">
-						<samp>
-							<Anchor
-								href="https://explorer.dusk.network/transactions/transaction?id={transaction.id}"
-								rel="noopener noreferrer"
-								target="_blank"
-							>
-								{middleEllipsis(
-									transaction.id,
-									calculateAdaptiveCharCount(screenWidth, 320, 640, 5, 20)
-								)}
-							</Anchor>
-						</samp>
-					</dd>
-					{#if transaction.tx_type}
-						<dt class="transactions-list__term">Type</dt>
+	{#await transactions}
+		<Throbber className="loading"/>
+	{:then transactionsResult}
+		<div class="transactions__lists">
+			{#if transactionsResult.length}
+				{@const sortedTransactions = transactionLimit
+					? getTransactionsShortlist(transactionsResult) : sortByHeightDesc(transactionsResult)}
+				{#each sortedTransactions as transaction (transaction.id)}
+					<dl class="transactions-list">
+						<dt class="transactions-list__term">Hash</dt>
+						<dd class="transactions-list__datum transactions-list__datum--hash">
+							<samp>
+								<Anchor
+									href="https://explorer.dusk.network/transactions/transaction?id={transaction.id}"
+									rel="noopener noreferrer"
+									target="_blank"
+								>
+									{middleEllipsis(
+										transaction.id,
+										calculateAdaptiveCharCount(screenWidth, 320, 640, 5, 20)
+									)}
+								</Anchor>
+							</samp>
+						</dd>
+						{#if transaction.tx_type}
+							<dt class="transactions-list__term">Type</dt>
+							<dd class="transactions-list__datum">
+								<Badge className="w-100" text={transaction.tx_type}/>
+							</dd>
+						{/if}
+						<dt class="transactions-list__term">Block</dt>
 						<dd class="transactions-list__datum">
-							<Badge className="w-100" text={transaction.tx_type}/>
+							{new Intl.NumberFormat(language).format(transaction.block_height)}
```

Comment:
why not using the `Badge` as before?

Reply:
That's just how GitHub displays the changes (in a not very nice way) – this hasn't changed, the Badge is used on line 81.